src/index.ts

```diff
@@ -1,105 +1,20 @@
-import * as nearAPI from "near-api-js";
 import { sha256 } from "js-sha256";
 
-import axios from "axios";
 import { v4 as uuidv4 } from "uuid";
 
-
-const MESSAGE_KEY = "calimeroSecret";
-const MESSAGE_HASH_KEY = "calimeroSecretHash";
 const AUTH_TOKEN_KEY = "calimeroToken";
-const CALIMERO_TOKEN_KEY = "caliToken";
-//Max valid period for a token would be 30 days
-export const MAX_CALIMERO_TOKEN_DURATION = 1000 * 60 * 60 * 24 * 30;
-
-export class WalletData {
```

Comment:
I wouldn't remove this but move it to another module. Dapps should be able to validate the token, also this library can be used in our backends.

Reply:
Should then all passed data from the url hash (#) be also saved in local storage under calimeroSecret and walletData ? Currently I only take 4 parameters from it..